source/Body.h

```diff
@@ -18,21 +18,27 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 #include "Angle.h"
 #include "Point.h"
+#include "SpriteParameters.h"
 
 #include <cstdint>
+#include <map>
 #include <string>
 
 class DataNode;
 class DataWriter;
 class Government;
 class Mask;
+class Outfit;
 class Sprite;
 
-
-
 // Class representing any object in the game that has a position, velocity, and
```

Comment:
As a general rule of thumb, dont touch what you dont have to ^^
```suggestion


// Class representing any object in the game that has a position, velocity, and
```